front-end/src/components/MessageForm.vue

```diff
@@ -272,14 +298,15 @@ const send = async () => {
   }
 
   // Đặt giá trị các biến về mặc định sau khi gửi
-  selectedPlatform.value = "";
-  selectedSender.value = "";
-  receiverURL.value = "";
-  selectedTemplate.value = "";
+  selectedPlatform.value = undefined;
```

Comment:
E nên gán giá trị tương ứng với type của biến thay vì undefined

Reply:
Vậy trong trường hợp em muốn nó là trạng thái không nhận giá trị nào thì sao ạ?

Reply:
> E nên gán giá trị tương ứng với type của biến thay vì undefined

Vậy trong trường hợp em muốn nó là trạng thái không nhận giá trị nào thì sao ạ?

Reply:
@namdnhn 
Thực ra thì dùng ntn cũng được nhưng theo a nghĩ thì nên dùng null
`Using undefined as an assignment value is generally discouraged in most programming languages because it can lead to confusion and unintended behaviors. In JavaScript, for example, undefined is a primitive value that represents an uninitialized or non-existent variable. Assigning undefined to a variable intentionally can make your code harder to understand and maintain.`